HTML/UraMasyu translated (日本語 — 裏ましゅ) (Y.Y.).html

```diff
@@ -0,0 +1,229 @@
+                      <path d="M1814.5 2343h5v590h-5v-590z"/>
+                      <path d="M1937.5 2933h-5v-590h5v590z"/>
+                      <path d="M2053 2458.5v5h-590v-5h590z"/>
+                      <path d="M1463 2581.5v-5h590v5h-590z"/>
+                      <path d="M2053 2694.5v5h-590v-5h590z"/>
+                      <path d="M1463 2817.5v-5h590v5h-590z"/>
+                      <path d="M1521 2402.5v-5h118v5h-118z" class="st1"/>
+                      <path d="M1636.5 2400h5v118h-5v-118z" class="st1"/>
+                      <path d="M1639 2515.5v5h-118v-5h118z" class="st1"/>
+                      <path d="M1523.5 2518h-5v-118h5v118z" class="st1"/>
+                      <path d="M1754.5 2518h5v354h-5v-354z" class="st1"/>
+                      <path d="M1757 2874.5v-5h236v5h-236z" class="st1"/>
+                      <path d="M1995.5 2872h-5v-354h5v354z" class="st1"/>
+                      <path d="M1872.5 2400h5v354h-5v-354z" class="st1"/>
+                      <path d="M1518.5 2636h5v236h-5v-236z" class="st1"/>
+                      <path d="M1521 2756.5v-5h118v5h-118z" class="st1"/>
+                    </g>
+                    <text class="st4" x="710.38" y="333.75">Allowed</text>
+                    <text class="st4" x="1760.6" y="337.75">Not Allowed</text>
+                    <text transform="translate(702.62 1207.36)"><tspan x="0" y="0" class="st5">Turn in white circles, and</tspan><tspan x="0" y="52.8" class="st5">pass straight through both</tspan><tspan x="0" y="105.6" class="st5">cells on either end.</tspan></text>
+                    <text transform="translate(1761.6 1207.36)"><tspan x="0" y="0" class="st5">No passing straight through</tspan><tspan x="0" y="52.8" class="st5">white circles or turning in</tspan><tspan x="0" y="105.6" class="st5">the cells on either end.</tspan></text>
+                    <text transform="translate(700 2098.63)"><tspan x="0" y="0" class="st5">Pass straight through black</tspan><tspan x="0" y="52.8" class="st5">circles, and turn in one or</tspan><tspan x="0" y="105.6" class="st5">both cells on either end.</tspan></text>
+                    <text transform="translate(1757.15 2098.63)"><tspan x="0" y="0" class="st5">No turning in black circles</tspan><tspan x="0" y="52.8" class="st5">or passing straight through</tspan><tspan x="0" y="105.6" class="st5">both cells on either end.</tspan></text>
+                    <text transform="translate(701 2994.68)"><tspan x="0" y="0" class="st5">The path must be a single,</tspan><tspan x="0" y="52.8" class="st5">continuous loop that passes</tspan><tspan x="0" y="105.6" class="st5">through all of the circles.</tspan></text>
+                    <text transform="translate(1757 2994.68)"><tspan x="0" y="0" class="st5">No small loops (top left),</tspan><tspan x="0" y="52.8" class="st5">spurrs (bottom left), or</tspan><tspan x="0" y="105.6" class="st5">trapped ends (right).</tspan></text>
```

Comment:
Making an imported SVG with lots of text that has been converted to <path> is not acceptable as it cannot be translated easily, and such path conversions take up hundreds more kilobytes than pure text. So I converted your diagrams to be embedded in the HTML and have actual text. Now you just have to type in the Japanese text you want here.